core/src/components/reorder/readme.md

```diff
@@ -13,8 +13,12 @@ Reorder is a component that allows an item to be dragged to change its order. It
 </ion-item>
 ```
 
-The position of the 
+## Properties
 
+| Property   | Attribute  | Description                            | Type      |
+| ---------- | ---------- | -------------------------------------- | --------- |
+| `slot`     | `slot`     | If `start`, the reorder icon will be shown before the label. If `end`, the reorder icon will be show after the label | `string` |
+| `name`     | `name`     | The name of the icon to use for the reorder handle. | `string` ||
```

Comment:
I don't think this needs to be here - it isn't a property of reorder but rather the icon.